test/jdk/java/util/concurrent/ConcurrentHashMap/MapLoops.java

```diff
@@ -45,6 +45,13 @@
  * @run main/timeout=1600 MapLoops
  */
 
+/*
+ * @test
+ * @summary Exercise multithreaded maps, using only heavy monitors.
+ * @library /test/lib
+ * @run main/othervm/timeout=1600 -XX:+IgnoreUnrecognizedVMOptions -XX:+UseHeavyMonitors MapLoops
```

Comment:
Did you want to also add -XX:+VerifyHeavyMonitors to this test?

Reply:
If UseHeavyMonitors only works properly on x86_64 you will need an `@requires` to restrict this test run to that platform.